src/gui/src/scriptWidget.cpp

```diff
@@ -376,12 +376,6 @@ class ScriptWidget::GuiSink : public spdlog::sinks::base_sink<Mutex>
 
       widget_->addLogToOutput(formatted_msg, msg_color);
     }
-
-    // process widget event queue, if main thread will process new text,
-    // otherwise there is nothing to process from this thread.
-    if (QThread::currentThread() == widget_->thread()) {
-      QCoreApplication::sendPostedEvents(widget_);
```

Comment:
@maliberty How about if I checked for the XDG error message and skipped updates only for that message which is causing constipation?

Reply:
I'm ok with that.  Those messages are pretty useless anyways